packages/ui/Onboarding/components/PathwaySelect.tsx

```diff
@@ -0,0 +1,41 @@
+interface PathwaySelectProps {
+  setselectedPathway: (pathway: "link" | "in-app" | null) => void;
+}
+
+export default function PathwaySelect({ setselectedPathway }: PathwaySelectProps) {
+  return (
+    <div className="space-y-16 text-center">
+      <div className="space-y-4">
+        <p className="text-4xl font-medium">How would you like to start?</p>
+        <p>You can always use both types of surveys</p>
+      </div>
+      <div className=" flex space-x-8">
+        <div
+          className="animation flex h-96 w-80 flex-col items-center justify-center rounded-2xl border border-slate-300 bg-white p-3 shadow-lg transition ease-in-out hover:scale-105"
+          onClick={() => {
+            // finishOnboarding();
+            localStorage.setItem("isNewUser", "true");
+            setselectedPathway("link");
+          }}>
+          <div className="h-full w-full rounded-xl bg-gray-500">Image</div>
+          <div className="my-4">
+            <p className="text-lg font-medium">Link Surveys</p>
+            <p className="text-xs">Create a new survey and share it via link.</p>
+          </div>
+        </div>
+        <div
+          className="flex h-96 w-80 flex-col items-center justify-center rounded-2xl border border-slate-300 bg-white  p-3 shadow-lg transition ease-in-out hover:scale-105"
```

Comment:
card code redundant, please refactor. we will use this kind of card a lot more going forward, pls add to UI package.

I've done it bc I was getting tired of making all changes on the cards twice.

Asking GPT to "refactor folllwing all major coding best practices" led to quite a good outcome:

`interface PathwaySelectProps {
  setselectedPathway: (pathway: "link" | "in-app" | null) => void;
}

// Define a type for the pathway options to ensure type safety and reusability
type PathwayOptionType = "link" | "in-app";

interface PathwayOptionProps {
  title: string;
  description: string;
  onSelect: () => void;
}

// Reusable PathwayOption component for each option
const PathwayOption: React.FC<PathwayOptionProps> = ({ title, description, onSelect }) => (
  <div
    className="flex h-96 w-80 cursor-pointer flex-col items-center justify-center rounded-2xl border border-slate-300 bg-white p-3 shadow-lg transition ease-in-out hover:scale-105"
    onClick={onSelect}
    role="button" // Improve accessibility
    tabIndex={0} // Make it focusable
  >
    <div className="h-full w-full rounded-xl bg-gray-500">Image</div>
    <div className="my-4 space-y-2">
      <p className="text-xl font-medium text-slate-800">{title}</p>
      <p className="text-sm text-slate-500">{description}</p>
    </div>
  </div>
);

export default function PathwaySelect({ setselectedPathway }: PathwaySelectProps) {
  // Helper function to handle selection
  const handleSelect = (pathway: PathwayOptionType) => {
    localStorage.setItem("isNewUser", "true");
    setselectedPathway(pathway);
  };

  return (
    <div className="space-y-16 text-center">
      <div className="space-y-4">
        <p className="text-4xl font-medium text-slate-800">How would you like to start?</p>
        <p className="text-sm text-slate-500">You can always use both types of surveys.</p>
      </div>
      <div className="flex space-x-8">
        <PathwayOption
          title="Link Surveys"
          description="Create a new survey and share a link."
          onSelect={() => handleSelect("link")}
        />
        <PathwayOption
          title="In-app Surveys"
          description="Run a survey on a website or in-app."
          onSelect={() => handleSelect("in-app")}
        />
      </div>
    </div>
  );
}
`